debuggers/gdb/gdbdebugger.js

```diff
@@ -60,8 +58,20 @@ define(function(require, exports, module) {
                     ["autoshow", "true"]
                 ]);
             });
-
+            
+            // must register ASAP, or debugger won't be ready for reconnects
             debug.registerDebugger(TYPE, plugin);
+
+            // writeFile root is workspace directory, unless given ~
+            var shimPath = "~/bin/c9gdbshim.js";
```

Comment:
I am looking for a way to make writefile call on demand, and only when missing
would it be ok to write this to `~/.c9/bin/c9gdbshim-<md5>.js` instead, or is this supposed to be used from other places too?